### PR TITLE
Fix loading page

### PR DESCRIPTION
**Pull request by Byasar3**

### CODE REVIEW

File: src/components/Loading/Loading.tsx
Comment (JoshS1234) [inline src/components/Loading/Loading.tsx:1]: Don't think setLoading should be in the dependency array, as I don't think it is a value that can change so can't trigger a rerender? Maybe check this with Ollie

Comment (JoshS1234) [inline src/components/Loading/Loading.tsx:1]: Return a spinner or something if loading is true (instead of null)

Comment (JoshS1234) [inline src/components/Loading/Loading.tsx:1]: remove empty div return and replace with just an empty return

**Review by JoshS1234 — CHANGES_REQUESTED** [review]
A few changes to make, Nabeel used a spinner for his Flappy bird so he might be able to suggest how to add that

File: src/components/Loading/Loading.tsx
Comment (Byasar3) [reply src/components/Loading/Loading.tsx:1]: test to pass the merge didn't run if this wasn't in the dependency array

Comment (JoshS1234) [reply src/components/Loading/Loading.tsx:1]: Ask Ollie about this I reckon

**Review by JoshS1234 — APPROVED** [review]
All good

File: src/App.tsx [hunk 1]
```diff
@@ -14,46 +14,58 @@ import Account from "./pages/Account/Account";
...
 import Dashboard from "./pages/Dashboard/Dashboard";
 import Create from "./pages/Create/Create";
 
-
 const App = () => {
   const { isAuthenticated, isAdmin } = useAuth();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setLoading(true);
```
Comment (olirob93) [inline src/App.tsx:27]: do we need this line, as we are already setting loading to true in the state by default

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ const ErrorPage = () => {
         LOOKS LIKE YOU'VE TAKEN A WRONG TURN.
       </h1>
       <ErrorSVG />  
-      <Link to={isAuthenticated ? "/" : "/auth"}>
+      <Link to={isAuthenticated ? "/" : "/"}>
```
Comment (olirob93) [inline src/pages/ErrorPage/ErrorPage.tsx:17]: can't this just be put to / now as that check is saying if true send to / or send to / if not ?

**Review by olirob93 — CHANGES_REQUESTED** [review]
A couple of things but looks good

File: src/pages/ErrorPage/ErrorPage.tsx (hunk 2 above)
Comment (JoshS1234) [reply src/pages/ErrorPage/ErrorPage.tsx:17]: Changed this now

File: src/App.tsx (hunk 1 above)
Comment (JoshS1234) [reply src/App.tsx:27]: Don't think so

**Review by olirob93 — APPROVED** [review]
all good

**Pull request merged**